src/univers/versions.py

```diff
@@ -343,3 +344,86 @@ def __gt__(self, other):
         if not isinstance(other, self.__class__):
             return NotImplemented
         return gentoo.vercmp(self.value, other.value) > 0
+
+
+@attr.s(frozen=True, order=False, eq=False, hash=True)
+class LegacyOpensslVersion(Version):
+    @classmethod
+    def is_valid(cls, string):
+        return OpensslVersion.is_valid_legacy(string)
```

Comment:
Implement this here and call it from OpensslVersion (ie the other way round)

Reply:
Yeah that makes more sense 😆